lib/api.js

```diff
@@ -1,6 +1,6 @@
 module.exports = adminsApi
 
-var clone = require('lodash.clone')
+var clone = require('lodash')
```

Comment:
here it is: https://github.com/hoodiehq/pouchdb-admins/issues/11

we still need this to be `var clone = require('lodash/clone')`, otherwise things will fall apart if `options.admins` is set to the [factory method](https://github.com/hoodiehq/pouchdb-admins#factory)